src/launch.py

```diff
@@ -71,6 +69,13 @@ def main():
        action='store_true',
        help="Load Qt's QAbstractItemModelTester into data models "
        '(requires Qt 5.11+)')
+    parser.add_argument('-d', '--debug', action='store_true',
+        help='Enable debugging output')
+    parser.add_argument('--debug-file', action='store_true',
+        help='Debugging output (logfile only)')
+    parser.add_argument('--debug-console', action='store_true',
+        help='Debugging output (console only)')
+    # Hidden processing for short-form '-d' synonym for --debug
```

Comment:
I have absolutely no idea what this comment means, as it's clearly not correct. I think it snuck in as part of a bad conflict merge or something.
```suggestion

```